src/NexusMods.Paths/Size.cs

```diff
@@ -69,45 +73,35 @@ namespace NexusMods.Paths;
     public static Size TB => FromLong(1024L * 1024 * 1024 * 1024);
 
     /// <inheritdoc />
-    public static Size operator /(Size left, double right) => From((ulong)(left._value / right));
+    public static Size operator /(Size left, double right) => From((ulong)(left.Value / right));
 
     /// <inheritdoc />
-    public static Size operator *(Size left, double right) => From((ulong)(left._value * right));
+    public static Size operator *(Size left, double right) => From((ulong)(left.Value * right));
 
     /// <inheritdoc />
-    public override string ToString() => _value.ToFileSizeString();
+    public override string ToString() => Value.ToFileSizeString();
 
     /// <inheritdoc />
     public static Bandwidth operator /(Size left, TimeSpan right)
     {
-        return Bandwidth.From((ulong)(left._value / right.TotalSeconds));
+        return Bandwidth.From((ulong)(left.Value / right.TotalSeconds));
     }
 
     /// <inheritdoc />
-    public static Size operator +(Size left, Size right) => From(left._value + right._value);
-
-    /// <inheritdoc />
-    public static Size operator -(Size left, Size right) => From(left._value - right._value);
-
-    /// <inheritdoc />
-    public static double operator /(Size left, Size right) => (double)left._value / right._value;
-
-    /// <inheritdoc />
-    public static bool operator >(Size left, Size right) => left._value > right._value;
+    public static Size operator +(Size left, Size right) => From(left.Value + right.Value);
 
     /// <inheritdoc />
-    public static bool operator >=(Size left, Size right) => left._value >= right._value;
+    public static Size operator -(Size left, Size right) => From(left.Value - right.Value);
 
     /// <inheritdoc />
-    public static bool operator <(Size left, Size right) => left._value < right._value;
+    public static double operator /(Size left, Size right) => (double)left.Value / right.Value;
 
-    /// <inheritdoc />
-    public static bool operator <=(Size left, Size right) => left._value <= right._value;
```

Comment:
What is the reason for removing these? Unneeded, unused, unsupported?

Reply:
`TransparentValueObjects` implements these already.